### PR TITLE
IPv6 support for inspector

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by dtantsur** [comment]
Why split out `ironic-common` here?

Additionally, I couldn't find where you actually source that file.

**Comment by stbenjam** [comment]
> Why split out ironic-common here?

It's the same as what's in ironic-image, I figured doing it the same way would be easier to port changes between the two repos. I could include it in the run inspector script if you'd prefer.

> Additionally, I couldn't find where you actually source that file.

That would help, wouldn't it? Fixed

File: runironic-inspector.sh [hunk 1]
```diff
@@ -31,10 +26,9 @@ mkdir -p /shared/log/ironic-inspector
 
 cp $CONFIG $CONFIG.orig
 
-crudini --set $CONFIG ironic endpoint_override http://$PROVISIONING_IP:6385
-crudini --set $CONFIG service_catalog endpoint_override http://$PROVISIONING_IP:5050
+crudini --set $CONFIG ironic endpoint_override http://$IRONIC_HOST_URL:6385
+crudini --set $CONFIG service_catalog endpoint_override http://$IRONIC_HOST_URL:5050
```
Comment (dtantsur) [inline runironic-inspector.sh]: Is it `IRONIC_HOST_URL` or `IRONIC_URL_HOST`?

Comment (stbenjam) [reply runironic-inspector.sh]: Thanks, nice catch - it's fixed.

**Review by dtantsur — APPROVED** [review]

**Review by juliakreger — APPROVED** [review]
/lgtm

**Pull request merged**